src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -258,15 +259,13 @@ export default [
             type === CONTEXT_MENU_TYPES.REPORT_ACTION && !ReportActionsUtils.isReportActionAttachment(reportAction) && !ReportActionsUtils.isMessageDeleted(reportAction),
 
         // If return value is true, we switch the `text` and `icon` on
-        // `ContextMenuItem` with `successText` and `successIcon` which will fallback to
+        // `ContextMenuItem` with `successText` and `successIcon` which will fall back to
         // the `text` and `icon`
         onPress: (closePopover, {reportAction, selection}) => {
             const isTaskAction = ReportActionsUtils.isTaskAction(reportAction);
-            const isCreateTaskAction = ReportActionsUtils.isCreatedTaskReportAction(reportAction);
             const isReportPreviewAction = ReportActionsUtils.isReportPreviewAction(reportAction);
             const message = _.last(lodashGet(reportAction, 'message', [{}]));
-            const reportID = lodashGet(reportAction, 'originalMessage.taskReportID', '').toString();
-            const messageHtml = isTaskAction || isCreateTaskAction ? Task.getTaskReportActionMessage(reportAction.actionName, reportID, isCreateTaskAction) : lodashGet(message, 'html', '');
+            const messageHtml = isTaskAction ? Task.getTaskReportActionMessage(reportAction) : lodashGet(message, 'html', '');
```

Comment:
Hi, it looks like this method was moved to `TaskUtils` but this reference wasn't updated. It results in the following webpack warning on `main` and it will probably cause the app to crash:

```
[web-server] WARNING in ./src/pages/home/report/ContextMenu/ContextMenuActions.js 277:37-68
[web-server] export 'getTaskReportActionMessage' (imported as 'Task') was not found in '@userActions/Task' (possible exports: canModifyTask, cancelTask, clearOutTaskInfo, clearOutTaskInfoAndNavigate, clearTaskErrors, completeTask, createTaskAndNavigate, dismissModalAndClearOutTaskInfo, editTask, editTaskAssignee, getAssignee, getShareDestination, getTaskAssigneeAccountID, reopenTask, setAssigneeValue, setDescriptionValue, setDetailsValue, setShareDestinationValue, setTaskReport, setTitleValue)
[web-server]  @ ./src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js 24:0-78 39:8-40 65:44-62
[web-server]  @ ./src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js 11:0-72 311:38-65
[web-server]  @ ./src/Expensify.js 43:0-108 209:39-69
[web-server]  @ ./src/App.js 23:0-36 55:97-106
[web-server]  @ ./index.js 6:0-28 11:9-12
```